hermes/models.py

```diff
@@ -54,24 +54,36 @@ class Target(models.Model):
 
 
 class NonLocalizedEvent(models.Model):
+    class NonLocalizedEventType(models.TextChoices):
```

Comment:
I look forward to one day adding Meteorite distribution fields or NEO position error ellipses to this :D